src/Pages/NBALiveStats.js

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState } from 'react';
+import '../App.css';
+import '../Styles/NBALiveStats.css';
+import Navbar from '../Components/LandingNavbar/Navbar'
+
+function NBALiveStats() {
+  const [playerStats, setPlayerStats] = useState([]);
+  const [searchQuery, setSearchQuery] = useState('');
+  const [filteredPlayerStats, setFilteredPlayerStats] = useState([]);
+
+  useEffect(() => {
+    // Simulating API call
```

Comment:
Excellent commenting

Reply:
Also can be ignored, havent set up the actual API call yet. Will get to it when Rohan sends the information.